.gitignore

```diff
@@ -10,6 +10,7 @@ config.log
 config.status
 config.sub
 configure
+compile
```

Comment:
Hurray for the autotools periodically adding new files.

If someone wants to move us to Meson, I'd be very happy. 🙂